deployment/copilot/devDeploy.sh

```diff
@@ -65,21 +58,6 @@ while [[ $# -gt 0 ]]; do
       DESTROY_ALL_COPILOT=true
       shift # past argument
       ;;
-    --auth-header-value)
-      REPLAY_AUTH_HEADER="$2"
-      shift # past argument
-      shift # past value
-      ;;
-    --aws-auth-header-user)
-      REPLAY_AWS_AUTH_HEADER_USER="$2"
-      shift # past argument
-      shift # past value
-      ;;
-    --aws-auth-header-secret)
-      REPLAY_AWS_AUTH_HEADER_SECRET="$2"
-      shift # past argument
-      shift # past value
-      ;;
```

Comment:
Could you have the dev-deploy script take in the arguments to use for the replayer?  Something like "--extraFlags" and then let the developer tack on any auth flags or anything else that might be necessary.  As it is, it will be painful for a dev to play around w/ different options.

And I would say that none of the arguments are checked - which should be fine if this is a "developer" script

Reply:
I am a little conflicted with what we should do here. I don't want the additional tax of actually knowing what arguments the Replayer takes which makes something like "--extraFlags" seem preferrable, however this also becomes a balancing act of does my default command already have an argument that is being passed in with "--extraFlags", it seems it would be easier for the user to just update the command in the script to what they need.

What might be slightly convenient is to allow a user to specify all the arguments for the Replayer (not sure how useful this is) in the devDeploy script, or do something like "--replayerAuthFlags" which defaults to our normal setting but can be overriden with whatever other flags

Lmk your thoughts here